content/doc/addons/keycloak.md

```diff
@@ -0,0 +1,181 @@
+---
+type: docs
+title: Keycloak
+description: This add-on provides a one-click Keycloak single sign-on and identity and access management (IAM) solution based on existing Clever Cloud services and taking advantage of the *Please open-it* experiences, our Keycloak expert partner.
+tags:
+- addons
+keywords:
+- Keycloak
+- IAM
+- Security
+- Autentication
+- Identity
+- OAuth
+- OpenID
+aliases:
+- /doc/deploy/addon/Keycloak
+type: docs
+---
+
+Keycloak is an open-source identity and access management (IAM) solution that offers powerful authentication features for services and secure applications. Thanks to this add-on, you can deploy Keycloak in your organization in just one click. It leverages all Clever Cloud features such as monitoring, scalability, high availability SLA, etc.
+
+To fit the most common needs, the Keycloak add-on comes with batteries included: sensible defaults, easy migration with realms import/export, Keycloak Metrics, default theme, etc.
+
+We have partnered with [*Please Open-it*](https://please-open.it/) to develop this add-on and to offer the most efficient and straightforward Keycloak service. They are experts for many years and are a go-to reference for advanced deployments and consulting.
+
+## Technical preview? What does that mean?
+
+We are proud to release great products, fit for the industry, and the easiest to use. To do that, we need your [feedback](https://www.clever-cloud.com/fr/contact-sales/). We also want to be sure we have worked in a good direction. Hence, we have to fire-test the product. That is why the Keycloak add-on is first **offered as a Tech Preview**, before releasing it for production use.
+
+During the Tech Preview period:
+- Features consistencies are battle-tested and may evolve; 
+- UX and interface are not final and may also evolve;
+- Bugs and potential issues feedback are deeply sought;
+- Features and pricing may evolve;
+- 0 € service & maintenance fees.
+
+## Key features
+Keycloak on Clever Cloud allows you to effortlessly set up a tailored authentication and access management solution, that can be adjusted to your needs and workloads. It offers a wide panel of services such as:
+- Secure Identity Management;
+- Single Sign-On (SSO);
+- Centralized Administration;
+- Customizable Authentication (OAuth 2.0, LDAP, SAML, OpenID Connect, and more...);
+- Easily load plugins and themes;
+- Multi Realms;
+- Monitoring and Logging;
+- Export and Import;
+- Support and Maintenance.
+
+## Need expert advice?
+The Clever Cloud Keycloak add-on is designed to meet the most common needs. However, realizing how central and critical authentication needs are for companies, we decided to tackle the issue and recommend expert advice for the most complex and loaded systems.
```

Comment:
This art belongs to a blog post. Maybe add `read more about our partnership with Please Open It` and a link toward the post once the blog post is published. 

```suggestion
The Clever Cloud Keycloak add-on is designed to meet the most common needs. For the most complex and loaded systems, contact  [*Please Open-it*](https://please-open.it/), our partners and Keycloak experts behind this add-on: [Contact them directly for more information](mailto:sales@clever-cloud.com).
```

Reply:
I rephrase it. Thank you for the advice